docs/getting-started/android-integration.md

```diff
@@ -0,0 +1,243 @@
+}
+```
+
+and in `app/build.gradle`:
+
+```gradle
+apply plugin: "org.mozilla.components.nimbus-gradle-plugin"
+
+nimbus {
+    // The path to the Nimbus feature manifest file
+    manifestFile = "nimbus.fml.yaml"
+
+    // Map from the variant name to the channel as experimenter and nimbus understand it.
+    // If nimbus's channels were accurately set up well for this project, then this
+    // shouldn't be needed.
+    channels = [
+            debug: "debug",
+            nightly: "nightly",
+            beta: "beta",
+            release: "release",
+    ]
+    // This is generated by the FML and should be checked into git.
+    // It will be fetched by Experimenter (the Nimbus experiment website)
+    // and used to inform experiment configuration.
+    experimenterManifest = ".experimenter.yaml"
+}
+```
+
+In this case, it should generate a file named in the `nimbus.fml.yaml` file. In the case of Fenix, this is called `FxNimbus`.
+
+# The start-up sequence
+
+Before using Nimbus in your Android app, you need to start it.
+
+The Nimbus SDK is a configuration store, making configuration available to the any thread, and — to a first approximation— to be immutable within the same session of the app.
+
+For this reason, we want to be start the Nimbus SDK as close to the beginning of the start of the app as possible.
+
+In Firefox for Android and Focus for Android, this is done at the beginning of the `Application#onCreate()` method.
+
+```kotlin
+class MyApplication: Application() {
+
+    lateinit var nimbus: NimbusApi
+
+    override fun onCreate() {
+        beginNimbusSetup()
+
+
+        // do the rest of the set up here.
+
+        finishNimbusSetup()
+    }
+
+    fun beginNimbusSetup() {
+        Megazord.init()
+
+        nimbus = createNimbus(this, NIMBUS_REMOTE_SETTINGS_ENDPOINT)
+    }
+
+    fun finishNimbusSetup() {
+        nimbus.fetchExperiments()
+    }
+
+    fun createNimbus(context: Context, urlString: String): NimbusApi {
+        val isAppFirstRun = context.settings().isFirstRun
+        val customTargetingAttibutes = JSONObject().apply {
+            // Put any custom attributes you want to use to segment an audience on to
+            // target your experiments.
+            put("is_first_run", isAppFirstRun)
+        }
+
+        val appInfo = NimbusAppInfo(
+            appName = "my-app-name",
+            channel = BuildConfig.BUILD_TYPE,
+            customTargetingAttributes = customTargetingAttributes
+        )
+
+        // Use the Nimbus builder to build a NimbusApi object.
+        return NimbusBuilder(context).apply {
+            url = urlString
+            errorReporter = { message, e ->
+                Logger.error("Nimbus error: $message", e)
+            }
+
+        }.build(appInfo)
+    }
+}
+```
+
+Notes:
+
+1. `Megazord.init()` is called before `createNimbus()`.
+1. `createNimbus` uses a `NimbusBuilder` to create the Nimbus object.
+1. Building a `JSONObject` of custom targeting attributes.
```

Comment:
I think the wording of this might be more clear by using the same verb tense as the second bullet
```suggestion
1. We build a `JSONObject` of custom targeting attributes.
```